docs/sphinx/source/whatsnew/v0.8.0.rst

```diff
@@ -44,6 +44,9 @@ Enhancements
   (:pull:`1017`)
 * Add :py:func:`pvlib.inverter.fit_sandia` that fits the Sandia inverter model
   to a set of inverter efficiency curves. (:pull:`1011`)
+* Added *racking_model*, *module_type*, and *temperature_model_parameters* to
```

Comment:
`*racking_model*` renders as italics on RTD.  Doesn't matter to me, just fyi if those were supposed to be inline literals (double backticks).

Reply:
I intended for them to be italics but can change them if someone cares. I think too many inline literals make the whats new extremely hard to read.